core/modules/user/user.install

```diff
@@ -1240,6 +1240,38 @@ function user_update_1019() {
   }
 }
 
+/**
+ * Make sure that all config translatables are set for user mails.
+ */
+function user_update_1020() {
```

Comment:
This code seems like duplicated from `system_update_1074()`. I think that we should have a function that does that, which we can then reuse here, and which contrib can also use.

Reply:
Adding a (reusable) function to core would take some time and might block this bugfix, possibly until May (1.19).

I'd prefer to get this bugfix in earlier.

Reply:
@klonos abstracting parts of these two functions provides almost no benefit since there's still a lot that would not be common. If there were 10 functions, then maybe, but we shouldn't try to abstract too early.